src/utils/mesh.js

```diff
@@ -62,113 +62,11 @@ function getCSaaSMeshConfig (core) {
 
 /**
  *
- * @param core
- * @param catalog
- * @param apiKey
+ * @param saas the saas endpoint
  */
-function getPaaSMeshConfig (core, catalog, apiKey) {
```

Comment:
Notably - the BYO now only works with SaaS endpoints. I took away ability to provision mesh for PaaS as it requires additional inputs.

We can add back if demand for this is found, otherwise for PaaS the idea would be for customer to manage that in the config file after choosing "demo".